android/build.gradle

```diff
@@ -1,12 +1,18 @@
 apply plugin: 'com.android.library'
 
+def DEFAULT_COMPILE_SDK_VERSION             = 25
+def DEFAULT_BUILD_TOOLS_VERSION             = "25.0.2"
+def DEFAULT_TARGET_SDK_VERSION              = 25
+def DEFAULT_GOOGLE_PLAY_SERVICES_VERSION    = "11.0.0"
+def DEFAULT_SUPPORT_LIB_VERSION             = "25.0.1"
+
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion rootProject.hasProperty('compileSdkVersion') ? rootProject.compileSdkVersion : DEFAULT_COMPILE_SDK_VERSION
+    buildToolsVersion buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : DEFAULT_BUILD_TOOLS_VERSION
```

Comment:
`buildToolsVersion` is duplicated here, please remove it.

Reply:
Removed, sorry about that!